src/EnergyPlus/BaseboardElectric.cc

```diff
@@ -361,24 +361,29 @@ namespace BaseboardElectric {
                                 "Baseboard Total Heating Energy",
                                 OutputProcessor::Unit::J,
                                 thisBaseboard.Energy,
-                                "System",
-                                "Sum",
+                                OutputProcessor::SOVTimeStepType::System,
+                                OutputProcessor::SOVStoreType::Summed,
```

Comment:
This is the kind of change in this PR -- lots of SetupOutputVariable calls changed from strings to the new enums.